onnxruntime/core/providers/cpu/nn/tfidfvectorizer.cc

```diff
@@ -141,12 +141,11 @@ struct TfIdfVectorizer::Impl {
   Impl(const Impl&) = delete;
   Impl& operator=(const Impl&) = delete;
 
-  void IncrementCount(size_t ngram_id, size_t row_num,
-                      std::vector<uint32_t>& frequencies) const {
+  void IncrementCount(size_t ngram_id, std::vector<uint32_t>& frequencies) const {
```

Comment:
suggest to make it a gsl::span<uint34_t> for more flexible output

Reply:
done